libs/ui-lib/lib/common/api/assisted-service/SupportedOpenshiftVersionsAPI.ts

```diff
@@ -2,12 +2,14 @@ import { client } from '../../api/axiosClient';
 import { OpenshiftVersion } from '@openshift-assisted/types/assisted-installer-service';
 
 const SupportedOpenshiftVersionsAPI = {
-  makeBaseURI() {
-    return `/v2/openshift-versions`;
+  makeBaseURI(latest_release: boolean) {
```

Comment:
Nit: our own variables should not use `snake_case`, since we can name them however we want. 
Suggestion would be `latestRelease` (for all new variables, like also in the `useOpenshiftVersions` hook)